collectd-couchdb/docs/counter.couchdb.couchdb.mrview.emits.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Mrview Emits
+brief: Number of invocations of `emit' in map functions in the view server
```

Comment:
@gautam-swaroop please change ` to '. ` are special characters in md and rst. This breaks the docs build.